avaframe/out3Plot/outAIMEC.py

```diff
@@ -23,7 +23,7 @@
 log = logging.getLogger(__name__)
 
 
-def visuTransfo(rasterTransfo, inputData, cfgSetup, pathDict, cfgFlags):
+def visuTransfo(rasterTransfo, inputData, cfgSetup, pathDict):
```

Comment:
Please add docstrings to the functions

Reply:
do you mean a more precise description of what the function does or do you also want the parameters...?

Reply:
description is fine, short and precise :-) 
I meant the parameters...